### PR TITLE
Add save/load comments in keras docs

### PR DESCRIPTION
**Pull request by dding3**
1. Add save/load comments in keras docs
2. fix https://github.com/intel-analytics/arda-docker/issues/566

### CODE REVIEW

File: docs/readthedocs/source/doc/DLlib/Overview/keras-api.md [hunk 1]
```diff
@@ -2831,3 +2831,45 @@ array([[[[0.43790358, 0.61913717, 0.2543214 ],
...
+import com.intel.analytics.bigdl.dllib.keras.Net
+
+val model = Net.load[Float]("/tmp/seq.model")
+```
+
+**Python:**
+```python
+from bigdl.dllib.net import Net
+Net.load("/tmp/seq.model")
+```
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/DLlib/Overview/keras-api.md:2875]: Need to make it consistent with Keras (https://www.tensorflow.org/api_docs/python/tf/keras/models/load_model, https://www.tensorflow.org/api_docs/python/tf/keras/models/save_model):

```python

model.save_model(...)

from bigdl.dllib.keras.models import load_model
model = keras.models.load_model("/tmp/seq.model")
```

```scala

model.saveModel(...)

import bigdl.dllib.keras.Models
model = Models.load_model("/tmp/seq.model")
```

**Comment by jason-dai** [comment]
You should split the PR into 2 (to fix 2 different problems)

File: python/dllib/src/bigdl/dllib/keras/models.py [hunk 2]
```diff
@@ -80,6 +80,12 @@ def from_jvalue(jvalue, bigdl_type="float"):
         model.value = jvalue
         return model
 
+    @staticmethod
+    def loadModel(model_path, bigdl_type="float"):
```
Comment (jason-dai) [inline python/dllib/src/bigdl/dllib/keras/models.py:84]: 1) `load_model`
2) `keras.models.load_model`, not `Model.load_model`

File: scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/keras/Model.scala [hunk 3]
```diff
@@ -292,4 +292,12 @@ object Model extends KerasLayerSerializable {
     Model(tGraph.inputs.toArray, new GraphRef(tGraph).getOutputs().toArray)
   }
 
+  def loadModel[T: ClassTag](path: String)(implicit ev: TensorNumeric[T]): Model[T] = {
```
Comment (jason-dai) [inline scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/keras/Model.scala:295]: `keras.Models.loadModel`

**Comment by dding3** [comment]
jekins pass. http://10.112.231.51:18888/view/ZOO-PR/job/BigDL-Orca-PR-Validation/288/
http://10.112.231.51:18888/view/ZOO-PR/job/BigDL-Orca-PR-Validation/287/
http://10.112.231.51:18888/view/ZOO-PR/job/BigDL-PR-Dllib-Validation/217/

File: scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/keras/models/Topology.scala [hunk 4]
```diff
@@ -1673,6 +1673,16 @@ object InternalDistriOptimizerV2 {
   }
 }
 
+object Models {
+  def loadModel[T: ClassTag](path: String)(implicit ev: TensorNumeric[T]): keras.Model[T] = {
+    val model = Net.load[T](path)
+    if (!model.isInstanceOf[Model[T]]) {
+      throw new RuntimeException("Not an Analytics Zoo Keras-style model.")
```
Comment (jason-dai) [inline scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/keras/models/Topology.scala:1680]: Why Analytics Zoo?

File: scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/keras/models/Topology.scala [hunk 5]
```diff
@@ -1673,6 +1673,16 @@ object InternalDistriOptimizerV2 {
   }
 }
 
+object Models {
+  def loadModel[T: ClassTag](path: String)(implicit ev: TensorNumeric[T]): keras.Model[T] = {
+    val model = Net.load[T](path)
+    if (!model.isInstanceOf[Model[T]]) {
```
Comment (jason-dai) [inline scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/keras/models/Topology.scala:1679]: Does it work for Sequential?

File: scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/keras/models/Topology.scala [hunk 6]
```diff
@@ -1673,6 +1673,16 @@ object InternalDistriOptimizerV2 {
...
+      throw new RuntimeException("Not an Analytics Zoo Keras-style model.")
+    }
+    model.asInstanceOf[keras.Model[T]]
```
Comment (jason-dai) [inline scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/keras/models/Topology.scala:1682]: Does it work for Sequential?

File: scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/keras/models/Topology.scala [hunk 7]
```diff
@@ -1673,6 +1673,16 @@ object InternalDistriOptimizerV2 {
   }
 }
 
+object Models {
+  def loadModel[T: ClassTag](path: String)(implicit ev: TensorNumeric[T]): keras.Model[T] = {
```
Comment (jason-dai) [inline scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/keras/models/Topology.scala:1677]: how about keras.Sequential?

Comment (dding3) [reply scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/keras/models/Topology.scala:1677]: Fixed in https://github.com/intel-analytics/BigDL/pull/3832

**Pull request merged**